src/components/Button.js

```diff
@@ -71,10 +73,13 @@ export const Button = ({
         !invert && isDelete && styles.rejectButton,
         big && [styles.bigButton, styles.bigButtonRadius],
         small && [styles.smallButton, styles.smallButtonRadius],
-        smallest && [styles.smallestButton, styles.smallestButtonRadius]
+        smallest && [styles.smallestButton, styles.smallestButtonRadius],
+        invert && lightest && [styles.buttonInvert, styles.lightestInvert]
       ]}
       containerStyle={[styles.container, needLandscapeStyle && styles.containerLandscape]}
-      ViewComponent={invert || isDelete || disabled ? undefined : DiagonalGradient}
+      ViewComponent={
+        invert || (invert && lightest) || isDelete || disabled ? undefined : DiagonalGradient
```

Comment:
i think this is not needed. if `invert` should be enough, right?

Reply:
yes